functions/src/handleAvailability.ts

```diff
@@ -0,0 +1,53 @@
+import { changeAvailability } from './adminUtils';
+import sendActionResponse from './sendActionResponse';
+
+const isAvailabilityText = (text: string) => {
+  return isMakeAvailable(text) || isMakeUnavailable(text);
+};
+
+const isMakeAvailable = (text: string) => {
+  return ['a', '-a', 'available'].includes(text);
+};
+
+const isMakeUnavailable = (text: string) => {
+  return ['u', '-u', 'unavailable'].includes(text);
+};
+
+const adjustAvailability = async (
+  text: string,
+  userName: string,
+  userId: string,
+  responseUrl: string
+) => {
+  if (isMakeAvailable(text)) {
+    await changeAvailability(userId, true);
+    await sendActionResponse(responseUrl, {
+      blocks: [
+        {
+          type: 'section',
+          text: {
+            type: 'mrkdwn',
+            text: `<@${userId}> is now available for reviews`,
+          },
+        },
+      ],
+      response_type: 'ephemeral',
+    });
+  } else if (isMakeUnavailable(text)) {
```

Comment:
you've already checked that your input is one of these two options, so the `else if` is kind of superfluous